src/main/java/com/minecolonies/coremod/colony/jobs/JobGuard.java

```diff
@@ -9,7 +9,9 @@
 import com.minecolonies.coremod.entity.ai.basic.AbstractAISkeleton;
 import com.minecolonies.coremod.entity.ai.citizen.guard.EntityAIMeleeGuard;
 import com.minecolonies.coremod.entity.ai.citizen.guard.EntityAIRangeGuard;
+import com.minecolonies.coremod.sounds.ArcherSounds;
 import com.minecolonies.coremod.sounds.DeliverymanSounds;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.sounds.DeliverymanSounds'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)